FrameMetadata/FrameMetadata/ViewController.m

```diff
@@ -0,0 +1,319 @@
+//
+//  ViewController.m
+//  Lets-Build-Publisher
+//
+//  Copyright (c) 2013 TokBox, Inc. All rights reserved.
+//
+
+#import "ViewController.h"
+#import <OpenTok/OpenTok.h>
+#import "TBExampleVideoCapture.h"
+#import "TBExampleVideoRender.h"
+
+@interface ViewController ()
+<OTSessionDelegate, OTSubscriberKitDelegate, OTPublisherDelegate, TBRendererDelegate>
+
+@end
+
+@implementation ViewController {
+    OTSession* _session;
+    OTPublisher* _publisher;
+    OTSubscriber* _subscriber;
+    
+    TBExampleVideoCapture* _defaultVideoCapture;
```

Comment:
why don't you use `@property` instead?

Reply:
I personally love using `@proprety`. Since the codebase has been declaring local variable this way. Should we consider to follow the way it is?

Reply:
I think it is a good point to start. Probably we would need to refactor our current samples to "modernize" them, but I think we can start with this one.